DirectEmployers/MyJobs
# [PD-2991] Sitemap URLs sans redirect.

## What's this PR do?
Links directly to a job, with all the redirect-y logic applied, in the sitemap itself.
This is the alternative to using redirects to get to the job.

## Where should the reviewer start?
If you're going to test manually, ensure you have jobs in solr and all of the `redirect_*` tables in place and some arbitrary data in them (specifically for view source 28).

## How should this be manually tested?
1. Add jobs to solr. 
2. Go to `http://site.jobs/sitemap-2017-02-27.xml` (changing the domain and date in the url as appropriate).
3. Confirm that the links provided (a) have all the good redirect-y stuff included and (b) go directly to the job without any redirects.
